CONN-4743 | Update ms-service-bus-connector.adoc (#680)
mulesoft/docs-connectors: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc

== Use Case: Get Queue list

> This use case retrieves the list of existing queues in a specific namespace. It returns the list in JSON format.

Reviewer: ```suggestion
This use case retrieves a list of existing queues in a specific namespace. It returns the list in JSON format.
```

image::ms-dynamics/ms-service-bus-flow.png[General]

> . Create a new Mule Application in Studio and select HTTP Listener as a source in the new flow.

Reviewer: ```suggestion
. Create a new Mule application in Studio and select *HTTP Listener* as a source in the new flow.
```
Is this a physical user selection in a GUI? I know you have some objection but I can't recall why....

. Add a new HTTP Listener configuration global element:
+

> . Specify the Host and Port parameters with the following values:

Reviewer: ditto: code parameters should be ` `; UI labels should be * *.
I will stop marking.

|Port |8081
|===
+
. Click *Save*.
. Assign the new global configuration to HTTP Listener.

> . Fill in the HTTP Listener path with the value `/servicebus`.

Reviewer: ```suggestion
. Complete the HTTP Listener path with the value `/servicebus`.
```

payload
----
+

> . Save and run the project as a Mule Application.

Reviewer: GLOBAL. No one else init. caps. this.
If you are representing a syntax value or UI label, then....
```suggestion
. Save and run the project as a Mule application.
```

> == Use Case: Windows Service Bus AMQP

Reviewer: Do not use punctuation in topic titles:
```suggestion
== Windows Service Bus AMQP Use Case
```

== Use Case: Windows Service Bus Management

> This use case contains flows that create topics, subscriptions, and rules.

Reviewer: ```suggestion
This use case contains flows that create topics, subscriptions, and rules:
```

* `service-bus-management-demoFlow`
+
Creates a topic by following these steps:

. Creates the topic description as specified and stores it as the payload.
. Creates the topic using the stored configuration from the previous step and the topic namespace, obtained from variable `topic.name`.

> . Builds the JSON for the topic.

Reviewer 2: ```
. Converts the newly created topic to a JSON object.
```

> . Logs the JSON.

Reviewer 2: ```
. Logs the JSON object.
```

+
* `service-bus-management-demoFlow1`
+
Creates a subscription by following these steps:

. Creates the subscription description as specified and stores it as the payload.
. Creates the subscription using the stored configuration from the previous step, the subscription name from the variable `subscription.name`, and the topic name from the variable `topic.name`.

> . Builds the JSON for the subscription.

Reviewer 2: ```
. Converts the newly created subscription to a JSON object.
```

> . Logs the JSON.

Reviewer 2: ```
. Logs the JSON object.
```

+
* `service-bus-management-demoFlow2`
+
Creates a rule by following these steps:

. Creates a rule with the specified configuration and stores it as the payload.
. Creates the rule using the stored configuration from the previous step, the rule name from the variable `rule.name`, the subscription name from the variable `subscription.name`, and the topic name from the variable `topic.name`.

> . Builds the JSON for the rule.

Reviewer 2: ```
. Converts the newly created rule to a JSON object.
```

> . Logs the JSON.

Reviewer 2: ```
. Logs the JSON object.
```
